src/groups/mqb/mqba/mqba_clientsession.cpp

```diff
@@ -747,14 +747,32 @@ void ClientSession::onHandleConfiguredDispatched(
     // PRECONDITIONS
     BSLS_ASSERT_SAFE(dispatcher()->inDispatcherThread(this));
 
+    bdlma::LocalSequentialAllocator<2048> localAlloc(d_state.d_allocator_p);
```

Comment:
Might be good to move this initialization back down, since there is a code path with early return where this allocator is not constructed (line 771).